src/promptflow-devkit/promptflow/_sdk/_tracing.py

```diff
@@ -522,8 +522,8 @@ def setup_exporter_to_pfs() -> None:
         _logger.info("tracer provider is set with resource attributes: %s", res.attributes)
     # set exporter to PFS
     # get OTLP endpoint from environment
-    endpoint = os.getenv(OTEL_EXPORTER_OTLP_ENDPOINT)
-    _logger.debug("environ OTEL_EXPORTER_OTLP_ENDPOINT: %s", endpoint)
+    endpoint = os.getenv(OTEL_EXPORTER_OTLP_TRACES_ENDPOINT)
+    _logger.debug("environ OTEL_EXPORTER_OTLP_TRACES_ENDPOINT: %s", endpoint)
```

Comment:
Do we need to have a fallback logic for the scenario that OTEL_EXPORTER_OTLP_ENDPOINT is set but OTEL_EXPORTER_OTLP_TRACES_ENDPOINT is not set?

Reply:
![image](https://github.com/microsoft/promptflow/assets/38847871/4ea65a37-251e-437a-b858-95b38f42f888)

this is how we handle the environ consumption, do you think it's enough or not?